### PR TITLE
Fix local build

### PR DESCRIPTION
**Pull request by VictorVelarde**
Some small tweaks I had to do to avoid build errors, when starting fresh in my local repo.

### CODE REVIEW

**Comment by VictorVelarde** [comment]
Without the explicit declaration of modules, my build failed consistently... 🤷‍♂

File: README.md [hunk 1]
```diff
@@ -8,3 +8,11 @@ This is a monorepo intended to contain all small-medium sized JS libraries that
 - A Metapackage that exposes all
 
 It uses [Lerna](https://lerna.js.org) to manage all the packages and inter-dependencies.
+
+Quick steps to check that build works correctly
+
+```sh
+ npm install
```
Comment (rjimenezda) [inline README.md]:
```suggestion
 nvm use
 npm install
```

File: packages/sql/index.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+declare module '@carto/toolkit-sql';
```
Comment (rjimenezda) [inline packages/sql/index.d.ts]: These declar should not be necessary, could you try removing them? And the references to index.d.ts

Comment (VictorVelarde) [reply packages/sql/index.d.ts]: I have removed them, and it compiles. But it complains on npm run test

**Pull request merged**